enhancements/network/ovn-hypershift-live-migration.md

```diff
@@ -0,0 +1,303 @@
+redirected over this host node. To do so the point to point routing has to be
+changed and to detect post live-migration event a pair of 
+kubevirt virt-launcher pod labels has to be inspected, `kubevirt.io/nodeName` 
+and `creationTimestamp`, the first one is set after live migration with the 
+name of the new host node and the second one allow ovn-k to disregard old
+virt-launcher pod (they are not deleted after live-migration).
+
+### Risks and Mitigations
+
+#### OVN extended arp_proxy features
+
+
+The current ovn arp_proxy is missing the following features: enforcing the mac 
+address and ipv6, ideally those get implemented and ovn-kubernetes have 
+them for 4.14 release.
+
+In case they don't arrive on time, it will not be able to configure IPv6 hosted
+clusters at hypershift and VMs arp table will need to be refreshed after live
+migration. 
+
+### Drawbacks
+
+Doing live migration over pod network with kubevirt and changing its topology may present some risks and need to be 
+thoroughly tested and supported.
+
+### Open Questions 
+
+Do this impact non hypershift payloads ? The control-plane will have some more
+load from the point to point routing but data-plane is not affected
+
+Do this work with future interconnect topology ? It's not using a distributed 
+switch per hosted cluster so that part should be fine, but it may needed 
+to replicate point to point routing at all node's ovn_cluster_router.
+
+### Test Plan
+
+Add some network tests to [hypershift conformance](https://github.com/openshift/origin/pull/27456)
+that will do live migration with the following scenarios:
+- TCP connection between VMs
+- TCP connection to VMs from service
+- TCP connection from VM to service
+
+The connection has to survive and downtime should be minimal.
+
+
+### Graduation Criteria
+
+At hypershift kubevirt provider GA this feature will be graduated since it unblocks the 
+live migration feature
+
+#### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+#### Tech Preview -> GA
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+- Document SLOs for the component
+- Conduct load testing
+- User facing documentation created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+#### Removing a deprecated feature
+n/a
+### Upgrade / Downgrade Strategy
+
+Since hypershift is tech preview before this enhancement Upgrade/Downgrade is 
+not supported for hosted clusters, but it should not break openshift 
+networking for other payloads on upgrade or downgrade.
+
+### Version Skew Strategy
+n/a
+
+### Operational Aspects of API Extensions
+
+#### Failure Modes
+
+Probable failure situations include
```

Comment:
another probable failure situation is leaked ip allocations due to bugs with this new feature, stale policies and routes left on ovn_cluser_router